--- a/feeds/packages/utils/dockerd/Makefile
+++ b/feeds/packages/utils/dockerd/Makefile
@@ -104,6 +104,9 @@ BUILDTAGS += selinux
 endif
 
 define Build/Compile
+	# Drop the host-binary copy step, it is useless when cross compiling
+	# and aborts the build if any of them is missing on the build host
+	$(SED) '/copy_binaries "/d' $(PKG_BUILD_DIR)/hack/make/binary-daemon
 	cd $(PKG_BUILD_DIR); \
 	$(GO_PKG_VARS) \
 	DOCKER_GITCOMMIT=$(PKG_GIT_SHORT_COMMIT) \
